SupportBank/Program.cs

```diff
@@ -1,12 +1,63 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Transactions;
 
 namespace SupportBank
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            
         }
     }
+
+    class Transaction
+    {
+        public Transaction(string date, string from, string to, string narrative, double amount)
+        {
+            Date = date;
+            From = from;
+            To = to;
+            Narrative = narrative;
+            Amount = amount;
+        }
+
+        public string Date { get;  }
+        public string From { get; }
+        public string To { get; }
+        public string Narrative { get; }
+        public double Amount { get; }
+    }
+
+    class Account
+    {
+        public Account(string name)
+        {
+            BalanceDue = 0.0;
+            Name = name;
+            Transactions = new List<Transaction>();
+
+        }
+        
+        public string Name { get; }
+        public double BalanceDue { get; set; }
+        public List<Transaction> Transactions { get; set; };
```

Comment:
I think these can have private `set` methods? It's good to avoid public `set`s whenever possible 🙂